### DISCUSSION TITLE
Remove “New” in “New text block”, “New redirect node”, “New data node”

### DISCUSSION THREAD

**Discussion by GrimPixel** [IndiePython/nodezator#65] [Minor issue]
The options with “New” and options without “New” all add new nodes. That “New” is not required.
I guess in earlier versions, the “New” options were placed elsewhere, so the word “New” was needed.
By the way, “Useful” in “Useful encapsulatios” can be removed as well, as they are all useful, which is the reason they are there.

**Comment by KennedyRichard** [comment]
Oh, those are great observations, so many unnecessary words.

This reminded me that a member of the community on discord also made similar remarks more than a year ago (https://discord.com/channels/980193047000272967/980193047461630113/1016964511636389909):

> In my opinion, the context menu can be cleaned up a bit by removing the words "New" and "node" before and after the name of every folder from a node pack. I think the node groups will be easier to find if they don't all start with New

On the occasion I made a list of such remarks but that one seems to have escaped me somehow. Thank you for bringing all of this to my attention.

I already finished all code changes for the next release, which should be out next week, but for such simple changes I'll make an exception and include them right away, specially since it has been so much time someone mentioned this.

**Reply by GrimPixel** [reply]
It's great that you take care of such details. These are the steps to exquisiteness.

**Comment by KennedyRichard** [comment]
I already performed these changes, so I'll be closing this discussion.

I didn't upload my changes to the online copy of my local repo because they will be pushed to GitHub and uploaded to PyPI where they wil lbe available to all users anyway, I expect, in couple of days.

Again, thank you for the feedback.
